documentation/src/main/asciidoc/topics/dependencies_maven/ispn_embedded.xml

```diff
@@ -1,5 +1,13 @@
-<dependency>
-  <groupId>org.infinispan</groupId>
-  <artifactId>infinispan-embedded</artifactId>
-  <version>${version.infinispan}</version>
-</dependency>
+<dependencyManagement>
+    <dependencies>
+       <dependency>
+           <groupId>org.infinispan</groupId>
+           <artifactId>infinispan-bom</artifactId>
+           <version>${version.infinispan}</version>
+       </dependency>
```

Comment:
you don't have to use the bom, but if you do, you need to import it:
```suggestion
       <dependency>
           <groupId>org.infinispan</groupId>
           <artifactId>infinispan-bom</artifactId>
           <version>${version.infinispan}</version>
           <type>pom</type>
           <scope>import</scope>
       </dependency>
```
you have an example in spring-boot repo: https://github.com/infinispan/infinispan-spring-boot/blob/master/pom.xml#L95